stdlib/3/hashlib.pyi

```diff
@@ -40,3 +40,39 @@ algorithms_available = ...  # type: AbstractSet[str]
 # New in version 3.4
 if sys.version_info >= (3, 4):
     def pbkdf2_hmac(hash_name: str, password: _DataType, salt: _DataType, iterations: int, dklen: Optional[int] = ...) -> bytes: ...
+
+if sys.version_info >= (3, 6):
+    class VarLenHash(metaclass=ABCMeta):
```

Comment:
This class doesn't actually exist as `hashlib.VarLenHash`, so it should be marked as private (`_VarLenHash`).

Looks like the same is true for Hash, but that's not new to this PR.